src/modules/ui.js

```diff
@@ -0,0 +1,19 @@
+import GetScore from './getFromAPi.js';
```

Comment:
I believe you have misspelled the path for this import and it is causing all sorts of errors as you can see in the screenshot below. ⬇️ Kindly fix this error. Except for this everything looks good.
![image](https://user-images.githubusercontent.com/33268927/212461762-dfff6945-c8c7-4b1d-8c1a-62f1b8302f0c.png)